ajax/libs/gif.js/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "gif.js",
+  "description": "JavaScript GIF encoding library",
+  "filename": "gif.min.js",
+  "author": "Johan Nordberg <code@johan-nordberg.com>",
+  "homepage": "http://jnordberg.github.io/gif.js/",
```

Comment:
Let's use `https`.
https://jnordberg.github.io/gif.js/
It's safety for browsing.
Thank you.